clang/docs/CMakeLists.txt

```diff
@@ -103,6 +103,13 @@ function (gen_rst_file_from_td output_file td_option source docs_targets)
 endfunction()
 
 if (LLVM_ENABLE_SPHINX)
+  llvm_find_program(dot)
```

Comment:
@Krishna-13-cyber, could we drop the `llvm_find_program(dot)` and rely on handling the `dot` discovery from llvm. This is what we do a few lines above for the case of doxygen.